cpp/common/env.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) ZeroC, Inc.
+
+#include <optional>
+#include <string>
+
+namespace common
+{
+    /// Get the value of an environment variable.
+    /// @param variableName The name of the environment variable.
+    /// @return The value of the environment variable, or nullopt if the variable is not set.
+    std::optional<std::string> getEnvironmentVariable(const char* variableName)
```

Comment:
I think it would be simpler to return a string, and not distinguish between "not set" and "set to empty".

Reply:
All the env APIs make this distinction. I don't find empty string to be any "simpler".

Reply:
See comment above.